src/commands/trialApp/removeTrialApp.ts

```diff
@@ -6,21 +6,32 @@
 import { MessageItem } from 'vscode';
 import { IActionContext } from 'vscode-azureextensionui';
 import { TrialAppContext } from '../../constants';
+import { ExpiredTrialAppTreeItem } from '../../explorer/trialApp/ExpiredTrialAppTreeItem';
+import { ITrialAppContext } from '../../explorer/trialApp/ITrialAppContext';
 import { TrialAppTreeItem } from '../../explorer/trialApp/TrialAppTreeItem';
 import { ext } from '../../extensionVariables';
 import { localize } from '../../localize';
 import { addTrialAppTelemetry } from './addTrialAppTelemetry';
 
-export async function removeTrialApp(context: IActionContext, node?: TrialAppTreeItem): Promise<void> {
+export async function removeTrialApp(context: IActionContext, node?: TrialAppTreeItem | ExpiredTrialAppTreeItem): Promise<void> {
     if (!node) {
         node = await ext.tree.showTreeItemPicker<TrialAppTreeItem>(TrialAppTreeItem.contextValue, context);
     }
 
-    const message: string = localize('removeTrialApp', 'Are you sure you want to remove trial app "{0}"?', node.client.fullName);
+    const trialAppContext: ITrialAppContext | undefined = ext.context.globalState.get(TrialAppContext);
```

Comment:
Rather than getting the context here, I think it makes sense to just use the label instead.  You have to have a context to create an ExpiredTrialApp anyway, so might as well remove this check.

Reply:
I mean changing `node.client.fullName` to just `node.label`.  I think that should be fine for this case since the label is the name.